engine/src/shell/platform/unity/darwin/ios/DeviceScreen.mm

```diff
@@ -0,0 +1,12 @@
+#import <UIKit/UIKit.h>
+#import "UIWidgetsDevice.h"
+extern "C"
```

Comment:
we should follow the code style in engine, e.g.,
(1) rename files to device_screen.mm, etc..
(2) use "UIWidgets_API" instead of "extern "C"" directly